app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @comment = @post.comment_entries.build
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comment_entries.build(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to post_path(@post), notice: 'Comment was successfully created.'
+    else
+      render :new
+    end
```

Comment:
You have done a great deal addressing all previously found issues, however, when a comment is created, it doesn't show on the UI. The normal behaviour should after a comment is created, the user should be redirected to the post page so the comment can reflect there, but when I create a comment and manually navigate to the posting page the comment is not there. I recommend addressing this problem in order to satisfy the requirement.

I have recorded a video to show this behaviour:

[Uploading screen-capture.webm…]()

Reply:
@DuaneDave Done with this change